src/logicblocks/event/store/store.py

```diff
@@ -48,7 +48,9 @@ def __aiter__(self) -> AsyncIterator[StoredEvent]:
         return self.iterate()
 
 
-class EventStream(EventSource[StreamIdentifier]):
+class EventStream[A: EventStorageAdapter = EventStorageAdapter](
```

Comment:
1. If you used the approach I mentioned above, you wouldn't need to expose `adapter` from `store`.
2. Why does this need to be added as a typevar?

Reply:
See comment above - it was a convenience to mean you don't need to hold a reference to the storage adaptor separately to the store

Reply:
Typevar means I can constrain the store type in `make_in_memory_event_broker`